tests/test_events/test_events_ratios.py

```diff
@@ -47,3 +51,25 @@ def test_event_checkin_and_cancellation_ratio_per_room(self):
             event_ratio_per_room_query,
             event_ratio_per_room_response
         )
+
+    def test_event_checkin_and_cancellation_single_room(self):
+        """
+        Test that an admin is able to get the ratio of checkins to bookings
+        for a single room
+        """
+        CommonTestCases.admin_token_assert_equal(
+            self,
+            event_ratio_single_room_query,
+            event_ratio_single_room_response
+        )
+
+    def test_event_checkin_cancellation_single_room_wrong_id(self):
```

Comment:
Same here

Reply:
My explanation above should suffice here also.